lesson_00/josephcaballero/README.MD

```diff
@@ -0,0 +1,27 @@
+# Joseph Caballero
+
+## Introduction
+    My name is Joseph Caballero, I started coding two years ago and always had an interest since my introduction to it. I was born in Hawaii and grew up primarily in New Jersey before moving to Delaware when I was about 9 or 10 years old.
```

Comment:
Remove the tab before the start of the paragraphs, it's causing GitHub to interpret this as code blocks and removes word wrapping which is weird and makes it harder to read. I'll allow you to keep it this way, but I'd highly recommend you change it.

Reply:
Removed and anything like it as well in the readme.md file just to be safe.